lib/AI-Agent.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ApplicationInsightsAgent>
-    <AgentLogger>INFO</AgentLogger>
```

Comment:
I know the AppInsights client library is performant but that would be much more logging we are sending so maybe we should verify the impact on DataStore performance?